src/components/Block/Space.vue

```diff
@@ -13,7 +13,10 @@ const { web3 } = useWeb3();
 const web3Account = computed(() => web3.value.account);
 
 const isAdmin = computed(() => {
-  const admins = props.space.admins.map(address => address.toLowerCase());
+  const admins = (props.space?.admins ?? []).map(address =>
```

Comment:
You can do this also. 

`props.space?.admins?.map(address => address.toLowerCase());`

and below use `admins?.includes()`

I think its always better to keep variables undefined unless there is a specific need for initialisation simply because undefined !== []

An empty array and undefined are supposed to be handled differently. In this case it doesn't matter. Just a good programming practice.